test/nodejs/index.js

```diff
@@ -131,6 +131,12 @@ test('constructor: throws if default_radius is not a number', function(assert) {
     assert.ok(new OSRM({algorithm: 'MLD', path: monaco_mld_path, default_radius: 'unlimited'}), 'Does accept unlimited');
 });
 
+test('constructor: takes a keepalive_timeout argument', function(assert) {
+    assert.plan(1);
+    var osrm = new OSRM({algorithm: 'MLD', path: monaco_mld_path, keepalive_timeout: 10});
```

Comment:
The Node bindings do not use the HTTP server, so the `keepalive_timeout` won't do anything here.
I would just remove the test.

Reply:
Sure, I just remove it.